obspy/core/trace.py

```diff
@@ -2597,7 +2597,7 @@ def remove_response(self, output="VEL", water_level=60, pre_filt=None,
             ax6 = fig.add_subplot(326, sharex=ax4)
             for ax_ in (ax1, ax2, ax3, ax4, ax5, ax6):
                 ax_.grid(zorder=-10)
-            ax1.text(0.05, 0.1, 'pre_filt: %s' % pre_filt,
+            ax1.text(0.05, 0.1, 'pre_filt: %s' % str(pre_filt),
```

Comment:
This is also done in #1219 and whichever gets merged last will thus require a rebase. Just something to keep in mind.